patch/llpcPatchInOutImportExport.cpp

```diff
@@ -30,6 +30,7 @@
  */
 #define DEBUG_TYPE "llpc-patch-in-out-import-export"
 
+#include "llvm/IR/IRBuilder.h"
```

Comment:
Remove this. Do not use IR builder for a simple case branch. Always use IR builder for the entire function and the entire file.

Reply:
Not using IRBuilder would be silly. It is much more convenient to use and gives more maintainable code. Also there is no convenient alternative to IRBuilder::CreateIntrinsic.

So, if you don't like mixed IRBuilder/non-IRBuilder code in the same function, I will separate my new code into two new functions for you.

Reply:
I don't object to using IR builder. I just want to avoid mixing the inconsistent codes because if someone wants to add more codes he might be confused about which style to choose.

I think it is unnecessary to make two functions for your codes since the processing of such built-ins are simple. Is it possible to move `IRBuilder<> builder(*m_pContext);` to the outside of the case branch. Maybe we have a chance to change the style of using IR builder for other case branches in this function?